lib/features.js

```diff
@@ -87,17 +87,28 @@ const features = {
         ],
         description: 'process TypeScript files with Babel'
     },
-    vue: {
+    vue2: {
         method: 'enableVueLoader()',
         // vue is needed so the end-user can do things
         // vue-template-compiler is a peer dep of vue-loader
         packages: [
-            { name: 'vue' },
-            { name: 'vue-loader', enforce_version: true },
+            { name: 'vue', version: '^2' },
+            { name: 'vue-loader', version: '^15' },
```

Comment:
New feature to force what version to show when we recommend something to the user. Causes `yarn add vue@^2` to be output.